lib/token-autocomplete.ts

```diff
@@ -733,7 +733,9 @@ class TokenAutocomplete {
                 me.loadSuggestions();
             });
             me.parent.textInput.addEventListener('focusout', function (event) {
-                me.hideSuggestions();
+                setTimeout(function () {
```

Comment:
Can you add a small comment explaining why a setTimeout is required here so it will not be removed later. And can you test if a setTimeout of 0 would also work? My first guess would be that it could as it puts the callback in the setTimeout to the end of the call stack.

Reply:
with 0 it never worked, with 100 it sometimes worked, with 200 it always worked :)

maybe we can improve this in the future, with some improvments in the click handler but for now i didn't see a good option, as we also get no new focus target in event, since it is only a click event on a object which doesn't gain focus